spec/acceptance/pulsar_hunters_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+
+describe 'Pulsar Hunters' do
+  let(:db)        { Sequel.sqlite }
+  let(:storage)   { Nero::Storage.new(db) }
+  let(:output)    { double("Output") }
+
+  let(:processor) do
+    Nero::Processor.new(storage, output, {
+      1224 => {
+        "algorithm" => "pulsar_hunters",
+        "gold_standard_limit" => 120,
+        "normal_limit" => 30
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.